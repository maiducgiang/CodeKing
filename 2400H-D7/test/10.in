10
28369 2700
7636 1079
12396 691
19398 2833
45027 2664
8596 994
8168 2618
13980 1957
10769 567
13472 429
