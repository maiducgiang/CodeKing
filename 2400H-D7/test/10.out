750676108
197252492
817923045
702550495
136190571
605001745
738731178
235255052
740682964
420226737
